File: slackapilibrary/src/main/java/jp/co/devjchankchan/slackapilibrary/LeaveSeatMonitoringListener.java
package jp.co.devjchankchan.slackapilibrary;

import org.altbeacon.beacon.Beacon;
import org.altbeacon.beacon.Region;

import java.util.Collection;

public interface LeaveSeatMonitoringListener {
    public void onUpdateRangeBeaconsInRegion(Collection<Beacon> beacons, Region region);
}

File: physicalbotlibrary/src/main/java/jp/co/devjchankchan/slackapilibrary/LeaveSeatMonitoringListener.kt
package jp.co.devjchankchan.slackapilibrary

import org.altbeacon.beacon.Beacon
import org.altbeacon.beacon.Region

interface LeaveSeatMonitoringListener {
    fun onUpdateRangeBeaconsInRegion(beacons: Collection<Beacon>, region: Region)
}
